### PR TITLE
Admin Registration

### PR DESCRIPTION
**Pull request by mich2635**
##### Admin Registration

### CODE REVIEW

**Comment by mich2635** [comment]
- [ ] Admin Registration is added
- [ ]

**Comment by mich2635** [comment]
- [x] Admin Registration Added

File: src/main/java/in/micheal/model/UserDetails.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
+package in.micheal.model;
+/**
+ * The information in this class is sensitive so 
+ * we used encapsulation
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/micheal/model/UserDetails.java]: Format the code.

File: src/main/java/in/micheal/service/AdminRegistration.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+	
+	
+	/**
+	 * This method is used to register admin Id in user details
+	 * @param adminObj
+	 * @return
+	 */
+	public static boolean adminRegistration(UserDetails adminObj)
+	{
+	boolean confirmation=false;
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/micheal/service/AdminRegistration.java]: Format the code.

File: src/main/java/in/micheal/validator/AdminRegistrationValidator.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+import in.micheal.model.*;
+
+/**
+ * This methods validate Admin Registration id ,
+ * it returns false if admin id already exists
+ * @author mich2635
+ *
+ */
+public class AdminRegistrationValidator extends AdminBookDebtuserDob {
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/micheal/validator/AdminRegistrationValidator.java]: Follow Naming Convention - AdminBookDebtuserDob

File: src/main/webapp/AdminRegistration.jsp [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+		if(errorMessage != null){
+			out.println("<font color='red'>" + errorMessage + "</font>");
+		}
+		%>
+	<form action="AdminRegistrationAction" method="get">
+	
+			<strong><label>ENTER YOUR ID :</label></strong><br>
+			<input type="number" required placeholder="ID" min=10000 max=10000000 autofocus name="adminId"><br>
+			<strong><label>ENTER YOUR PASSWORD :</label></strong><br>
+			<input type="password" required placeholder="PASSWORD" min=10000 max=10000000 name="adminPassword"><br>
```
Comment (csys-fresher-batch-2021) [inline src/main/webapp/AdminRegistration.jsp]: Why you have used min and max for password.?

File: src/main/webapp/AdminRegistration.jsp [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+			<strong><label>ENTER YOUR EMAIL</label></strong><br>
+			<input type="email" required placeholder="EMAIL"><br>
+			<strong><label>ENTER YOUR PHONE NO</label></strong><br>
+			<input type=number required placeholder="+91" min=999999999 max=9999999999><br><br>
```
Comment (csys-fresher-batch-2021) [inline src/main/webapp/AdminRegistration.jsp]: min  = 1000000000

File: src/main/webapp/AdminRegistration.jsp [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+			<button class="btn btn-primary">SUBMIT</button>&nbsp &nbsp &nbsp
```
Comment (csys-fresher-batch-2021) [inline src/main/webapp/AdminRegistration.jsp]: &nbsp;

File: src/main/webapp/index.jsp [hunk 7]
```diff
@@ -1,19 +1,15 @@
-<%@page import="java.util.Date"%>
-<%@page import="java.time.LocalDate"%>
-<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core"%>
-<%@ taglib prefix="fmt" uri="http://java.sun.com/jsp/jstl/fmt"%>
-<html>
+
+<!DOCTYPE html>
+<html lang="en">
 <head>
 <title>MyApp</title>
```
Comment (csys-fresher-batch-2021) [inline src/main/webapp/index.jsp]: Rename title

**Review by ghost — CHANGES_REQUESTED** [review]
Added review comments

File: src/main/java/in/micheal/service/AllDetailsDB.java [hunk 8]
```diff
@@ -0,0 +1,10 @@
+package in.micheal.service;
+
+import in.micheal.model.*;
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/micheal/service/AllDetailsDB.java:3]: Avoid using "*" in imports

File: src/main/webapp/AdminRegistration.jsp [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+		<main class="container-fluid">
+			<h3>ADMIN REGISTRATION</h3>
+			
+			<%
+		String errorMessage = request.getParameter("errorMessage");
+		if(errorMessage != null){
+			out.println("<font color='red'>" + errorMessage + "</font>");
+		}
+		%>
+	<form action="AdminRegistrationAction" method="get">
```
Comment (csys-fresher-batch-2021) [inline src/main/webapp/AdminRegistration.jsp:20]: method = "post"

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**